### PR TITLE
fix bug 41726: Scheduled Posts tab -> Publish Time is Not Based on th…

### LINKED ISSUE #41726: Scheduled Posts Tab: Publish Time is Not Based on the Site's Time Zone
#### Steps to reproduce

1. Ensure that the site's time zone is different than your computer's time zone.
2. Navigate to My Site > Site > Posts, click on "Add New Post".
3. Schedule the post to publish at some point in the future.
4. Navigate to My Site > Site > Posts > Scheduled

#### What I expected
The time shown in the Scheduled tab should be displayed in the same time zone as your site.

#### What happened instead
The time shown in the Scheduled tab is different than what is shown in the editor. The editor shows the time based on the site's time zone whereas the Scheduled tab shows it based on your devices' time zone.

#### Browser / OS version
Chrome 81 and macOS 10.15.4

#### Screenshot / Video
Screen Shot 2020-05-02 at 11 16 41
Screen Shot 2020-05-02 at 11 16 58

#### Context / Source

#user-report here: 2932498-zen

### PR DESCRIPTION
**Pull request by sserafini1**
#### Changes proposed in this Pull Request

Changed the time zone of the scheduled post

Fixes #41726 

#### Testing instructions

The time shown in the Scheduled tab should be displayed in the same time zone as your site.

### CODE REVIEW

**Comment by sserafini1** [comment]
@jsnajdr

File: client/my-sites/post-relative-time-status/index.jsx [hunk 1]
```diff
@@ -2,7 +2,6 @@
  * External dependencies
  */
 import PropTypes from 'prop-types';
-import { localize } from 'i18n-calypso';
```
Comment (Aurorum) [inline client/my-sites/post-relative-time-status/index.jsx:5]: I'm guessing that this is still a WIP, but `localize` applies here as it is necessary for the translatable strings within this file. 

Screenshot 2020-05-06 at 09 25 11

Comment (sserafini1) [reply client/my-sites/post-relative-time-status/index.jsx:5]: Thank you very much for answering!
You're right, I am a newbie =P

**Comment by bluefuton** [comment]
Hi @sserafini1! Thanks for getting involved with Calypso.

I can't see any changes in the branch that you've pushed most recently. Is there anything you'd like to push so we can review it?

**Comment by sserafini1** [comment]
> Hi @sserafini1! Thanks for getting involved with Calypso.
> 
> I can't see any changes in the branch that you've pushed most recently. Is there anything you'd like to push so we can review it?

Hello @bluefuton! I'm sorry, since I am looking at this issue I couldn't notice the bug 41726 in my code, everything works fine. I think that someone could have fixed it before I did the pull request, or I  might not be experienced enough with the product to see the problem, yet.

**Comment by bluefuton** [comment]
No worries @sserafini1! I'll close this pull request for now 👍

**Pull request closed without merging**